plugins/module_utils/gpg_utils.py

```diff
@@ -106,6 +106,10 @@ def __init__(
         self.file_extension = file_extension
         self.pass_gpg_id_file = pass_gpg_id_file
 
+        # Warn if running as root
+        if os.geteuid() == 0:
+            print("warn: running as uid 0 (root), ensure the correct gnupg keyring is loaded")
```

Comment:
This is *not* how to do it. Ansible communicates with it's modules via stdin/stdout, and has separate functionality for warnings